core-bundle/src/Resources/public/sectionwizard.js

```diff
@@ -0,0 +1,101 @@
+(function(){
+    'use strict';
+
+    const initialized = [];
```

Comment:
We should use a `WeakMap` here so that we don’t create a memory leak.
_(Probably not really relevant for the section wizard, but if this code gets reused for other widgets too it would make the implementation more robust IMO)_
```suggestion
    const initialized = new WeakMap();
```